### DISCUSSION TITLE
Submit job feature for parametric setup

### DISCUSSION THREAD

**Discussion by RyanAshbaugh** [ansys/pyaedt#3927] [Q&A] [answered]
Does PyAEDT have an implementation for using the submit job feature for a parametric setup rather than using m3d.analyze_setup() to run a simulation? This would be great in order to deal with limited licenses and running multiple simulations.

**Comment by Samuelopez-ansys** [comment, marked as answer]
@RyanAshbaugh PyAEDT has this method to submit job:

https://aedt.docs.pyansys.com/version/stable/API/_autosummary/pyaedt.desktop.Desktop.submit_job.html

and this one to submit to the Ansys cloud:

https://aedt.docs.pyansys.com/version/stable/API/_autosummary/pyaedt.desktop.Desktop.submit_ansys_cloud_job.html

It is a bit tricky, but by modifying the "Job_settings.areg" file you could set the job settings. If you help regarding job submission, please reach the Ansys technical support.

**Reply by RyanAshbaugh** [reply]
Awesome, thanks Samuel. 

Another question though, for the first one, the second parameter is the clustername. What is the default clustername? I intend to submit a job to run locally, and I can't seem to find something that lines up in AEDT regarding a clustername.

**Reply by RyanAshbaugh** [reply]
Actually looking at an example .areg file I think I see what the cluster name is now. A bit confusing because none of the Ansys options use the "cluster name" as far as I can tell, but that's alright.

**Reply by kinielsen** [reply]
Hi @RyanAshbaugh , 
Can you please explain where you found the Cluster Name in .areg file?

**Reply by RyanAshbaugh** [reply]
@kinielsen I think the best way is to save your current setup to a .areg file. I don't remember exactly where it is but there is an option to save the config to an .areg file somewhere. Then you will see in the .areg file when the default cluster name is being set to, and then use that.

**Reply by kinielsen** [reply]
Thanks @RyanAshbaugh . Seems to be a good solution!
